README.md

```diff
@@ -236,23 +278,23 @@ git push
-| --------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
-| `<PLATE>.csv.gz`                                          | Aggregated well-level profiles                                                                                          |
-| `<PLATE>_augmented.csv.gz`                                | Metadata annotated profiles                                                                                             |
-| `<PLATE>_normalized.csv.gz`                               | Profiles normalized to the whole plate                                                                                  |
-| `<PLATE>_normalized_negcon.csv.gz `                       | Profiles normalized to the negative control                                                                             |
-| `<PLATE>_normalized_feature_select_<LEVEL>.csv.gz`        | Whole plate normalized profiles that are feature selected at the `plate`,  `batch` or `all plates` level                |
-| `<PLATE>_normalized_feature_select_negcon_<LEVEL>.csv.gz` | Negative control normalized profiles that are feature selected at the `plate`,  `batch` or `all plates` level           |
-| `<BATCH>_normalized_feature_select_<LEVEL>.csv.gz`        | Batch level stacked whole plate normalized profiles that are feature selected at the `batch` or `all plates` level      |
-| `<BATCH>_normalized_feature_select_<LEVEL>.gct`           | `.gct` file created from the `<BATCH>_normalized_feature_select_<LEVEL>.csv.gz` file                                    |
-| `<BATCH>_normalized_feature_select_negcon_<LEVEL>.csv.gz` | Batch level stacked negative control normalized profiles that are feature selected at the `batch` or `all plates` level |
-| `<BATCH>_normalized_feature_select_negcon_<LEVEL>.gct`    | `.gct` file created from the `<BATCH>_normalized_feature_select_negcon_<LEVEL>.csv.gz` file                             |
-| `summary.tsv`                                             | Summary statistics                                                                                                      |
-| `<PLATE>_cell_count.png`                                  | Plate cell count                                                                                                        |
-| `<PLATE>_correlation.png`                                 | Pairwise correlation between all the wells on a plate                                                                   |
-| `<PLATE>_position_effect.png`                             | Percent Matching between each well and other wells in the same row and column                                           |
-	
+| Filename | Description | Location |
```

Comment:
I added in file Location because it makes it easier to know how to explicitly send files to DVC (since with DVC we have to send them explicitly and can't just push everything to Git and let Git LFS sort it out).

@niranjchandrasekaran 
- Am I correct that these paths are hardcoded in utils.py so I can be explicit about the paths for DVC instructions?
- Can you fill in the missing paths for me? (Are they just profiles/BATCH ?)

Reply:
@ErinWeisbart I somehow never got a notification and just happened upon this when I visited the repo. Just filled in the missing paths. Sorry for the delay.

> Am I correct that these paths are hardcoded in utils.py so I can be explicit about the paths for DVC instructions?
Yes they are hardcoded.

Reply:
Thanks @niranjchandrasekaran !